ansible/roles/test/files/ptftests/py3/copp_tests.py

```diff
@@ -352,10 +353,18 @@ def __init__(self):
         # Marvell based platforms have cir/cbs in steps of 125
         elif self.hw_sku in {"Nokia-M0-7215", "Nokia-7215", "Nokia-7215-A1"}:
             self.PPS_LIMIT = 250
+        # Cisco G100 based platform has CIR 600
+        elif self.asic_type == "cisco-8000" and "8111" in self.platform:
+            self.PPS_LIMIT = 600
         else:
             self.PPS_LIMIT = 100
-        self.PPS_LIMIT_MIN = self.PPS_LIMIT * 0.9
-        self.PPS_LIMIT_MAX = self.PPS_LIMIT * 1.3
+        if self.asic_type == "cisco-8000" and any(platform in self.platform
+                                                  for platform in ["8102", "8101", "8111"]):
+            self.PPS_LIMIT_MIN = self.PPS_LIMIT * 0.8
```

Comment:
why we need change here?

Reply:
In the investigation on the meter profiles and calculation on the profile parameters. Will update later.

Reply:
Removed this change. Instead I changed the CIR configuration.

In Q200 meter profiles, minimum CBS is 1024, and minimum EBS is 2048.
For 30 seconds' traffic, if CIR is 100, then the expected result is:
Green: 1024 + 100 * 30 = 4024
Yellow: 2048
Theoretical rate of punted packets: (4024 + 2048) / 30 = 202.4, which is outside of the range [0.9 * 100, 1.3 * 100].

To fall in the script pass criteria [0.9 * CIR, 1.3 * CIR], set the CIR 400 pps.
Green: 1024 + 400 * 30 = 13024
Yellow: 2048
Theoretical rate of punted packets: (13024 + 2048) / 30 = 502.4, which is < 400 * 1.3.